GoogleForCreators/web-stories-wp
# Templates: Update Fitness Template Aspect Ratio to 9:16

## Summary
Updates Fitness template to new aspect ratio and fixes other critiques put in latest template design pass for fitness.

Designs & Critiques:
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61

## Relevant Technical Choices
NA

## To-do
~Still need a couple background images that have the color/blend-mode baked in. made notes of this in figma~

## User-facing changes
Fitness template update

## Testing Instructions
Go to explore templates and view the Fitness template. Each page should now match these designs:
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61

---



Fixes #2473

## Screenshots
<details>
<summary>Click to expand</summary>
<img width="264" alt="Screen Shot 2020-06-19 at 3 16 48 PM" src="https://user-images.githubusercontent.com/35983235/85180521-ec47c900-b240-11ea-95b3-d4f571afaf25.png">
<img width="264" alt="Screen Shot 2020-06-19 at 3 16 01 PM" src="https://user-images.githubusercontent.com/35983235/85180524-eeaa2300-b240-11ea-89fa-d1ca1a900740.png">
<img width="264" alt="Screen Shot 2020-06-19 at 3 16 40 PM" src="https://user-images.githubusercontent.com/35983235/85180525-ef42b980-b240-11ea-92b5-362d6b852249.png">
<img width="267" alt="Screen Shot 2020-06-19 at 3 16 07 PM" src="https://user-images.githubusercontent.com/35983235/85180526-efdb5000-b240-11ea-83e8-5bb245ef9928.png">
<img width="264" alt="Screen Shot 2020-06-19 at 3 16 13 PM" src="https://user-images.githubusercontent.com/35983235/85180529-f10c7d00-b240-11ea-946b-a48f85c7a95c.png">
<img width="262" alt="Screen Shot 2020-06-19 at 3 16 21 PM" src="https://user-images.githubusercontent.com/35983235/85180530-f10c7d00-b240-11ea-9ff7-acfca7bd0bcb.png">
<img width="265" alt="Screen Shot 2020-06-19 at 3 16 28 PM" src="https://user-images.githubusercontent.com/35983235/85180533-f10c7d00-b240-11ea-970b-d4ec536bb229.png">
<img width="266" alt="Screen Shot 2020-06-19 at 3 16 34 PM" src="https://user-images.githubusercontent.com/35983235/85180535-f1a51380-b240-11ea-8e63-b58af7e95fbb.png">
</details>
